cupyx/scipy/ndimage/filters.py

```diff
@@ -62,6 +63,93 @@ def convolve(input, weights, output=None, mode='reflect', cval=0.0, origin=0):
                                   True)
 
 
+def correlate1d(input, weights, axis=-1, output=None, mode="reflect", cval=0.0,
+                origin=0):
+    """One-dimensional correlate.
+    The array is correlated with the given kernel.
```

Comment:
Please add blank lines between the summary and the description as in the other CuPy docstrings

Reply:
Completed in #3390